src/main/java/com/smartbear/readyapi/plugin/git/ui/LocalRepositoryForm.java

```diff
@@ -23,7 +23,7 @@ public CredentialsProvider getCredentialsProvider() {
     @Override
     public JPanel getComponent() {
         JPanel panel = new JPanel();
-        panel.add(new JLabel("There is a local repository in the Project Directory"));
+        panel.add(new JLabel("Specify the path to the local repository in the Project Directory field."));
```

Comment:
It should be ``Specify a path to a local repository in the Project Directory field.`` We don't mean any specific repository it can be any local repository.